### PR TITLE
Feature/doc update for v1

### PR DESCRIPTION
**Pull request by gangtao**
some update based on feedbacks from jove

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -52,60 +50,43 @@ import timeplus_client
...
+api_address = os.environ.get("TIMEPLUS_HOST")
+worksapce = os.environ.get("TIMEPLUS_WORKSAPCE")
 
-# Configure API key authorization: ApiKeyAuth
+# Configure API key and address
 configuration = timeplus_client.Configuration()
-configuration.api_key['X-Api-Key'] = 'YOUR_API_KEY'
-# Uncomment below to setup prefix (e.g. Bearer) for API key, if needed
-# configuration.api_key_prefix['X-Api-Key'] = 'Bearer'
+configuration.address(api_address)
```
Comment (jovezhong) [inline client/README.md:69]: Better to mention the `api_address` has to be start with `http` or `https`

File: client/timeplus/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "0.3.0"   # noqa: F401
+__version__ = "1.0.0"   # noqa: F401
```
Comment (jovezhong) [inline client/timeplus/version.py:1]: are we going to use 1.0.1 in this update?

File: client/timeplus_client/configuration.py [hunk 3]
```diff
@@ -249,3 +249,20 @@ def to_debug_report(self):
             "Version of the API: 1.0.0-oas3\n"
             "SDK Package Version: 1.0.0".format(env=sys.platform, pyversion=sys.version)
         )
+
+    def apikey(self, key):
+        self.api_key["X-Api-Key"] = key
```
Comment (jovezhong) [inline client/timeplus_client/configuration.py:254]: We maybe can check the length of the apikey, say 60 char. If the length is not 60, the api key must be invalid

**Review by jovezhong — APPROVED** [review]

File: client/timeplus/version.py (hunk 2 above)
Comment (gangtao) [reply client/timeplus/version.py:1]: that will be updated with git tag

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**